src/index.html

```diff
@@ -1,14 +1,192 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Human rights</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-<body>
-  <h1>Human rights</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <main>
+      <!-- <h1>Human rights</h1> -->
+      <h1 class="mainheader">Universal Declaration of Human Rights</h1>
+      <h2 class="articleheader">Preamble</h2>
+      <p class="paragraph">
+        Whereas recognition of the inherent dignity and of the equal and inalienable
+        rights of all members of the human family is the foundation of freedom, justice
+        and peace in the world,        <br>
```

Comment:
```suggestion
        and peace in the world, 
        <br>
```
Put `<br>` tags either on newlines, or right in the end (without so many spaces)